### ISSUE TITLE
Questions about val loss

### ISSUE THREAD

**Issue by LWYgut** [kaen2891/stethoscope-guided_supervised_contrastive_learning#2]
Hi @kaen2891 . Thank you for opensource code !
When I reproduced your code, I found that the validation curve kept increasing. I am very confused by this result and would like to ask if you have encountered this situation?
![image](https://github.com/kaen2891/stethoscope-guided_supervised_contrastive_learning/assets/134259528/e8fa7cac-7b1d-410f-b825-51f4cbf1ebbe)

Additionally, I followed your code and the score I obtained was (Best Sp 86.64, Se 36.45, Sc 61.54 when Epoch=18). Are these situations normal? The S_p value is 36.45, which is a little lower than the reported value in the paper of 43.55 ± 5.93.
![image](https://github.com/kaen2891/stethoscope-guided_supervised_contrastive_learning/assets/134259528/cd2fe925-311a-4d77-8d60-0b91b9b5156b)

Thank you very much !

**Comment by kaen2891** [comment]
Hi @LWYgut 

1) I will check the training/val losses by each epoch and will upload it here soon.

2) As we mentioned in the paper, these results can be different for each seed. So we obtained the mean values of five seeds, as we coded in a script file (seed= 1 2 3 4 5). Could you check each seed's results, and let me know? Further, these results can be different by GPU or Cuda version.

Thanks

**Comment by kaen2891** [comment]
Hi @LWYgut

Sorry for late reply. Our team also addressed this issue in here from [your question ](https://github.com/raymin0223/patch-mix_contrastive_learning/issues/7)and I think this is same reason. Hopefully, this response address your question.

**Comment by LWYgut** [comment]
Hi @kaen2891 . 

Ok. Apologies for my late reply. Thank you for your patient explanation.
